### ISSUE TITLE
Staging Review finding: Time selections does not behave as expected

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128362]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments Team
**Product:** Appointments
**Feature:** Oracle Health Scheduling - Direct Schedule and Request for an Appointment

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When navigating the time options for a date with a screen reader, the screen reader announcement is confusing and the behavior is not as expected. 

When the time radios are navigated via arrow keys, the announcement is: "Friday 8:30am option selected radio button not checked". The "option selected" text is coming from the component:
```
<span class="vads-u-visibility--screen-reader">8:30 <span aria-hidden="true">a.m.</span> <span class="sr-only">AM</span> option selected</span>
```

Besides the opposing announcement, the behavior doesn't align to user expectation either. Typically when navigating radio groups, the radio buttons select as arrow navigation occurs. In this component, the radio selection does not happen on arrow navigation, which may confuse users. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="400" height="915" alt="radio selection issue" src="https://github.com/user-attachments/assets/32e6e9b4-3d44-47df-8c35-409227a19d82" />

</details>

## Recommended action

Ensure that the state of the component matches the actual state, and ensure that components follow expected behavior - e.g. radio buttons are selected as they receive focus via arrow navigation. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG22/#name-role-value)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@jasonday can you retest this? A fix went in recently that may have resolved it
